File: data/src/main/java/com/github/gmetal/data/entity/SessionEntity.java
package com.github.gmetal.data.entity;

import com.bluelinelabs.logansquare.annotation.JsonField;
import com.bluelinelabs.logansquare.annotation.JsonObject;

@JsonObject
public class SessionEntity extends BaseMoviesDbEntity {

    @JsonField
    public boolean success;

    @JsonField(name = "session_id")
    public String sessionId;
}

File: data/src/main/java/com/github/gmetal/data/entity/SessionEntity.kt
package com.github.gmetal.data.entity

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
class SessionEntity : BaseMoviesDbEntity() {

    var success: Boolean = false

    @SerialName("session_id")
    var sessionId: String? = null
}
